### DISCUSSION TITLE
Physical Size of Monitor

### DISCUSSION THREAD

**Discussion by Rewrite4828** [slint-ui/slint#3271] [Q&A] [answered]
Hello,

Another beginner question: Is there a way to get the screen size in cm? This is because sometimes, on high-resolution monitors, the text and components show up as extremely small (which puts strain on the eyes).  I would like to avoid this.

Regards

**Comment by ogoffart** [comment, marked as answer]
We don't expose this, so you'll have to use native API to get the DPI.

What we expose is the scale factor which should be set automatically to a sensible value depending on the screen dpi.

If you want to make things bigger, the best is to adjust the scale factor.
